setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='wordcloud',
-    version='1.1.3',
```

Comment:
Haha, this was just a habit. Sorry, I always increase the version before I add a new feature.
Shall I just reset it?